### PR TITLE
API applied

### PR DESCRIPTION
**Pull request by Mekuseo**
- Create a new game with the name of your choice by using the API.
- Make sure that you saved the ID of your game that will be returned by API.
- Implement the "Refresh" button that will get all scores for a game created by you from the API (receiving data from the API and parsing the JSON).
Implement the form "Submit" button that will save a score for a game created by you (sending data to the API).
- Use arrow functions instead of the function keyword.
- Use async and await JavaScript features to consume the API.

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @Mekuseo 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] Kindly make the following changes to your README.md file
- Include a **_Future Features_** Section with 1 -3 features that can be added to the project in the future
- Remove the Microverse logo. If you have yours, you can replace it with your logo
- The Project name should be the name of this project and not the _**[your_project_name]**_

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -1 +1,78 @@
...
+//       headers: {
+//         'Content-Type': 'application/json',
+//       },
+//       body: JSON.stringify({
+//         user: name,
+//         score,
+//       }),
+//     },
+//   );
+// };
```
Comment (GedeonTS) [inline src/index.js]: - [x] Kindly make sure to remove all commented out lines of code to make your project look professional :recycle:

File: src/index.js [hunk 2]
```diff
@@ -1 +1,78 @@
...
+
+refreshBtn.addEventListener('click',
+  getScores);
+
+submitBtn.addEventListener('click', async (event) => {
+  event.preventDefault();
+  getScores();
+  // window.location.reload();
```
Comment (GedeonTS) [inline src/index.js]: - [x] Good job posting the data to the API when the user submit :+1:  Kindly make sure to remove this commented line to make your code cleaner :+1:

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @Mekuseo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- [x] Good use of ECS6 syntax :+1: 
- [x] Posting and Getting scores successfully :+1: 
- [x] No linter errors :+1:    
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] I suggest writing more professional commit messages, by starting with a capital letter and using the imperative form to describe clearly what change you just made :recycle: 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (Mekuseo) [reply src/index.js]: Thank you for the review. Done

File: src/index.js (hunk 1 above)
Comment (Mekuseo) [reply src/index.js]: Thank you for the review. Done

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Mekuseo 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**